index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="css/styles.css" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500&display=swap" rel="stylesheet">
+  <title>Yiwen’s Project1</title>
+</head>
+
+<body>
+
+  <header>
+    <h3 class="j1">Eile mit Weile</h3>
+    <h3 class="j2">Zeit fur Performance</h3>
+  </header>
+
+  <main>
+    <h1 class="i1">Mai</h1>
+    <h2 class="i4 twentynineteen">2019</h2>
+
+    <ul class="a4">
+      <li>a&a</li>
+      <li>Sophie Germanier</li>
+      <li>Gisela Hochuli</li>
+      <li>Anne-Sophie Mlamali</li>
+      <li>Patrizio Welti</li>
+      <li>Mahtola Wittmer</li>
+    </ul>
+
+    <div class="a6">
+      <ul>
+        <li>laura Breitschmid &</li>
+        <li>Eva-Maria Knusel</li>
+      </ul>
+      <h2>4</h2>
+    </div>
+
+    <ul class="b1">
+      <li>Bujar Berisha</li>
+      <li>Kim Gvozdic</li>
+      <li>Chris Hunter</li>
+      <li>Pascal Lampert</li>
+      <li>Alexandra Meyer &</li>
+      <li>Sevenn Perrig</li>
+    </ul>
+
+    <div class="b3">
+      <ul>
+        <li>Rochus Lussi</li>
+        <li>Heinz Stahlhut</li>
+      </ul>
+      <h2>5</h2>
+    </div>
+
+    <div class="c1">
+      <ul>
+        <li>Claudia Kubler</li>
+      </ul>
+      <h2 class="nine">9</h2>
+    </div>
+
+    <div class="c4">
+      <ul>
+        <li>Claudia Grimm &</li>
+        <li>darts</li>
+      </ul>
+      <h2 class="ten">10</h2>
+    </div>
+
+    <ul class="c6">
+      <li>Bebjarmin Heller</li>
+      <li>Anna-Sabina Zurrer</li>
+    </ul>
+
+    <div class="d3">
+      <ul>
+        <li>Leo Hofmann &</li>
+        <li>Benjamin van Bebber</li>
+      </ul>
+      <h2 class="eleven">11</h2>
+    </div>
+
+    <ul class="e2">
+      <li>Daniel Haller</li>
+      <li>Monica Germann</li>
+      <li>Maryina Lussi</li>
+    </ul>
+
+    <div class="e6">
+      <ul>
+        <li>Angela Anzi</li>
+        <li>Andrea Saemann</li>
+      </ul>
+      <h2 class="twelve">12</h2>
+    </div>
+
+    <div class="f1">
+      <ul>
+        <li>Bettina Diel</li>
+      </ul>
+      <h2 class="sixteen">16</h2>
+    </div>
+
+    <div class="f3">
+      <ul>
+        <li>Jens Nielsen</li>
+        <li>Judith Keller</li>
+      </ul>
+      <h2 class="seventeen">17</h2>
+    </div>
+
+    <ul class="f5">
+      <li>Rplf Hermann</li>
+    </ul>
```

Comment:
Since, in this case, it's not really a list, I wonder if you could have gotten away with just a `div`:
```suggestion
    <div class="f5">Rplf Hermann</div>
```